plugins/woocommerce-admin/client/core-profiler/style.scss

```diff
@@ -344,3 +345,124 @@
 		margin-top: 20px;
 	}
 }
+
+// Business Info Page
+
+.woocommerce-profiler-business-information {
+	display: flex;
+	flex-direction: column;
+	.woocommerce-profiler-business-information__content {
+		max-width: 615px;
+		width: 100%;
+		display: flex;
+		align-self: center;
+
+		& > div {
+			width: 100%;
+		}
+
+		.woocommerce-profiler-business-information-form {
+			display: flex;
+			flex-direction: column;
+			justify-content: space-between;
+			height: 100%;
+			max-width: 404px;
+			width: 100%;
+		}
+
+		.woocommerce-profiler-question-label,
+		.woocommerce-profiler-business-info-store-name
+		.components-base-control__label {
+			text-transform: uppercase;
+			color: $gray-900;
+			font-weight: 500;
+			font-size: 11px;
+			line-height: 16px;
+			margin: 0 0 8px;
+		}
+
+		.woocommerce-profiler-question-label
+		.woocommerce-profiler-question-required {
+			color: #cc1818;
+			padding-left: 3px;
+		}
+
+		.woocommerce-profiler-business-info-store-name
+		.components-text-control__input {
+			height: 40px;
+			border-color: #bbb;
+			border-radius: 2px;
+			border-width: 1px;
+		}
+
+		.woocommerce-profiler-select-control__industry {
+			margin-bottom: 20px;
+		}
+
+		.woocommerce-profiler-question-subtext {
+			color: $gray-700;
+			font-size: 12px;
+			line-height: 16px;
+			font-weight: 400;
+			margin-top: 0;
+			margin-bottom: 20px;
+		}
+
+		.woocommerce-select-control__control-icon {
+			display: none;
+		}
+
+		.woocommerce-select-control__control.is-active {
+			.components-base-control__label {
+				display: none;
+			}
+		}
+
+		.woocommerce-select-control.is-searchable
+		.woocommerce-select-control__control-input {
+			margin: 0;
+			padding: 0;
+		}
+
+		.woocommerce-select-control.is-searchable
+		.components-base-control__label {
+			left: 13px;
+		}
+
+		.woocommerce-profiler-geolocation-notice {
+			margin: 0;
+			margin-bottom: 32px;
+			padding-right: 0;
+
+			@include breakpoint( '<782px' ) {
+				margin-bottom: 100px;
+			}
+
+			.components-notice__dismiss > svg {
+				height: 16px;
+				fill: #000;
+			}
+
+			.components-notice__dismiss {
+				height: 16px;
+				margin-right: 8px;
+			}
+
+			p {
```

Comment:
It seems that the notice text color is different and there is too much padding/margin between the country selecter and notice. 😅 

![Screenshot 2023-05-25 at 18 18 18](https://github.com/woocommerce/woocommerce/assets/4344253/91c3c668-fe1c-4d28-8ac2-d7a1fefb3b63)

Expected:

![Screenshot 2023-05-25 at 18 18 28](https://github.com/woocommerce/woocommerce/assets/4344253/d676f244-e4ae-4276-afce-950c1b33e26a)

Reply:
Sharp eyes 👀 
Fixed in https://github.com/woocommerce/woocommerce/pull/38412/commits/4c3e694332bde603017409c84ae2966f1785aa94